src/components/standard/Standard.java

```diff
@@ -0,0 +1,16 @@
+package components.standard;
+
+/**
+ * Standard interface for OSU component design.
+ *
+ * @param <T>
+ *            The type of the component
+ */
+public interface Standard<T> {
```

Comment:
This file doesn't need to exist if you add the OSU jar to the lib folder. Otherwise, you should add all of the standard methods here.